src/objective-c/change-comments.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
+
+import re
+import sys
+
+
+if len(sys.argv) < 2:
+  print("Please provide at least one source file name as argument.")
+  quit()
```

Comment:
I just Googled how to exit a script in Python and saw it said `quit()` was preferable to `sys.exit()` as it wouldn't exit the interpreter. It doesn't seem to require any import and it does work.